packages/databricks-vscode/src/extension.ts

```diff
@@ -170,6 +179,23 @@ export async function activate(
         `${path.delimiter}${context.asAbsolutePath("./bin")}`
     );
 
+    const cliDeps = getCLIDependenciesEnvVars(context);
+    for (const [key, value] of Object.entries(cliDeps)) {
```

Comment:
The idea is to always use the packaged tf binaries? Can you add a comment for this?

Reply:
Yes, we always use bundled dependencies, this approach seems to be the simples one. I've added the comment here